src/OrleansAzureUtils/Providers/Streams/AzureQueue/AzureQueueAdapterFactory.cs

```diff
@@ -29,6 +29,11 @@ public class AzureQueueAdapterFactory : IQueueAdapterFactory
         /// <summary>"DeploymentId".</summary>
         public const string DeploymentIdPropertyName = "DeploymentId";
 
+        /// <summary>
+        /// Application level failure handler override.
+        /// </summary>
+        protected Func<QueueId, Task<IStreamFailureHandler>> StreamFailureHandlerFactory { private get; set; }
```

Comment:
This is required for the azure queue tests.  I disabled the test for other reasons, I'll open an issue about soon, but code is related.